packages/@aws-cdk/aws-ecs/test/fargate/fargate-task-definition.test.ts

```diff
@@ -241,8 +242,92 @@ describe('fargate task definition', () => {
         taskDefinition.taskRole;
       }).toThrow('This operation requires the taskRole in ImportedTaskDefinition to be defined. ' +
         'Add the \'taskRole\' in ImportedTaskDefinitionProps to instantiate ImportedTaskDefinition');
+    });
+
+
+    test('runtime testing', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      new ecs.FargateTaskDefinition(stack, 'FargateTaskDefWindows', {
+        cpu: 1024,
+        memoryLimitMiB: 2048,
+        runtimePlatform: {
+          operatingSystemFamily: OperatingSystemFamily.WINDOWS_SERVER_2019_CORE,
+          cpuArchitecture: CpuArchitecture.X86_64,
+        },
+      });
+
+      new ecs.FargateTaskDefinition(stack, 'FargateTaskDefGraviton2', {
```

Comment:
Can you separate the Graviton part to it's own test?

Reply:
okay, no problem. 👍